### DISCUSSION TITLE
Angular - Erweiterbarkeit der Komponente Table um Eingabefelder

### DISCUSSION THREAD

**Discussion by BenRichter** [public-ui/kolibri#1296] [Q&A] [answered]
Hallo,

wir möchten in der Table-Komponente Daten direkt editierbar machen. Es sollen also Eingabefelder in manchen Spalten verfügbar sein und eventuell auch Buttons o.ä.
Aus der Dokumentation erschließt sich jedoch aktuell noch nicht, ob und wie man in der Tabelle andere Elemente einbringen kann, außer Texte. Gibt es dafür Slots oder kann man die Struktur flexibler hinterlegen, als über das _data Attribut?

Vielen Dank!

**Comment by deleonio** [comment]
Hallo @AndreasZimnas,

könnt Ihr vielleicht bei der Frage unterstützen. 

Viele Grüße
Martin

**Comment by deleonio** [comment, marked as answer]
Hier ein kleines Beispiel zum Weiterentwickeln: https://codesandbox.io/s/angular-table-cell-rendering-g6c4nk

Im Cell-Renderer bekommen wir den Table-Cell-Node und können dann mit jeweiligen Framework wieder einzelne Standalone Components rendern?!

@web-dave oder @Thomas-Schmidt kannst Du hier vielleicht weiterhelfen?!

**Reply by deleonio** [reply]
Hallo @BenRichter,

hast Du für uns ein Beispiel, wie Ihr Angular-Bindings innerhalb der Zellen nutzen könnt?

Vielen Dank und beste Grüße
Martin

**Reply by web-dave** [reply]
Moin.

Ist das ein open source project?

Aus der Codesandbox werde ich nicht schlau in Bezug auf die Frage.

Könnt ihr die Frage mit etwas mehr Infos konkretisieren?
Wäre ja evtl ein gutes topic für einen Stream wenn OSS.

**Reply by deleonio** [reply]
Hi @web-dave ,

ich habe eben das Beispiel im CodeSandbox überschrieben, weil ich vergessen hatte es zu forken. Ich stelle es fix wieder her 🥵.

**Reply by deleonio** [reply]
Hallo Zusammen,

ich habe das Projekt wiederhergestellt. Es geht um **Spalte C**. Wie bekommen wir da Angular Components rein, mit denen wir interagieren können?

https://codesandbox.io/s/angular-table-cell-rendering-g6c4nk?file=/src/app/app.component.ts

**Reply by deleonio** [reply]
Hier ein React-Sample dazu: https://codesandbox.io/s/kolibri-react-table-cell-rendering-sample-0l4drc
